aspnetcore/mvc/razor-pages/index.md

```diff
@@ -148,6 +148,12 @@ The `Customer` property uses `[BindProperty]` attribute to opt in to model bindi
 
 Razor Pages, by default, bind properties only with non-GET verbs. Binding to properties can reduce the amount of code you have to write. Binding reduces code by using the same property to render form fields (`<input asp-for="Customer.Name" />`) and accept the input.
 
+> [!NOTE]
+> For security reasons, you must opt in to binding GET request data to page model properties. Proceed with caution when trusting user input and mapping it to properties.
```

Comment:
>Proceed with caution

That's not very helpful. How about something along the lines.

Verify user input.

Reply:
Should you add. You might need to bind GET to support searching - or something like that. Gives motivation.